ruby/lib/google/protobuf/repeated_field.rb

```diff
@@ -32,9 +32,26 @@
 module Google
   module Protobuf
     class RepeatedField
+      extend Forwardable
+
+      def_delegators :to_ary,
+        :bsearch, :compact, :fetch, :inspect,
```

Comment:
I like this, much cleaner! Just one thing though -- can we add a test or two to make sure that the forwarding works? We don't have to be exhaustive over all the forwarded methods; just a few (e.g. maybe `map`).